install/playbooks/roles/dovecot/templates/dovecot-ldap.conf.ext

```diff
@@ -41,8 +41,10 @@ dnpass = {{ lookup("password", ldap.roPasswdParams) }}
 # dn is still the logged in user. Normally you want to keep this empty.
 #sasl_authz_id =
 
-# Use TLS to connect to the LDAP server.
-tls = yes
+# Use TLS to connect to the LDAP server is not necessary,
+# as it is on the same server. Putting yes here also seems to require
+# additional permissions for certificates sharing by dovecot.
+tls = no
```

Comment:
Apparently, you already dealt with that in the past since the apparmor profile for `usr.lib.dovecot.auth` includes the ldap certificate:
```
$ git grep 'letsencrypt' install/playbooks/roles/dovecot/templates/apparmor.d/
install/playbooks/roles/dovecot/templates/apparmor.d/local/usr.lib.dovecot.auth:  /etc/letsencrypt/archive/ldap.{{ network.domain }}/* r,
install/playbooks/roles/dovecot/templates/apparmor.d/local/usr.lib.dovecot.config:  /etc/letsencrypt/archive/imap.{{ network.domain }}/* r,
install/playbooks/roles/dovecot/templates/apparmor.d/local/usr.lib.dovecot.config:  /etc/letsencrypt/archive/pop3.{{ network.domain }}/* r,
install/playbooks/roles/dovecot/templates/apparmor.d/local/usr.sbin.dovecot:  /etc/letsencrypt/archive/imap.{{ network.domain }}/* r,
install/playbooks/roles/dovecot/templates/apparmor.d/local/usr.sbin.dovecot:  /etc/letsencrypt/archive/pop3.{{ network.domain }}/* r,
```
Any idea why you would have activated TLS then?

Reply:
I don't understand why the dovecot need to access the LDAP certificate to establish a TLS connection.